### ISSUE TITLE
issue about learning rate, trainsize, deccay_rate,

### ISSUE THREAD

**Issue by StormArcher** [1900zpf/MVGNet#2]
Very nice work, Congratulation for TCSVT publish.

May I ask whether the training parameters are follow paper or code? 
because they are not consistent

1) paper version 
trainsize=352
deccay_rate=5e-4

1) code version 
trainsize=384
deccay_rate=0.1

**Comment by StormArcher** [comment]
the batchszie is 2?

**Comment by Otanh1900** [comment]
Hello, the training details should be based on what is provided in the paper. (The parameters in the code are from new experiments conducted during the revision process to address reviewers' comments.) Thank you for your interest in this work.
